InventoryAdminUi/view/adminhtml/ui_component/inventory_source_listing.xml

```diff
@@ -134,6 +134,18 @@
                 </editor>
             </settings>
         </column>
+        <column name="type_code" sortOrder="35">
```

Comment:
Should be remade after the SourceDataObject reverting.
You may see https://github.com/magento/inventory/blob/62f89e2f13f776cc6d30f97d4f2922e734ed3207/InventoryInStorePickupAdminUi/view/adminhtml/ui_component/inventory_source_listing.xml#L10 as an example